### ISSUE TITLE
Treemap Datalabel Overflow

### ISSUE THREAD

**Issue by pawanvjadhav** [highslide-software/highcharts.com#4305]
Is it possible to set text overflow behavior for Treemap data labels same as Axis Tick label ?
If axis tick label contains more characters than available space, highchart adds ellipsis.

**Comment by pawanvjadhav** [comment]
Any update ? Are you looking into this?

**Comment by pawanvjadhav** [comment]
Any update?

**Comment by jon-a-nygaard** [comment]
To add ellipsis with text overflow, add the following option to the series:
```javascript
dataLabels: {
  style: {
    textOverflow: "ellipsis"
  }
}
```
Also the SVG Renderer did not set the text-overflow correctly. This is fixed in next version 4.1.8. For those who are waiting for the update, or wants to use a lower version, here is a quickfix:
```javascript
// Add this code after Highcharts is loaded, and before creating the chart.
Highcharts.SVGElement.prototype.textProps.push("textOverflow");
```

Here is a demo: http://jsfiddle.net/jon_a_nygaard/so7uh1fe/
